### ISSUE TITLE
Make GAM available via winget

### ISSUE THREAD

**Issue by BiosPlus** [GAM-team/GAM#1673]
I've currently made the effort to submit a PR to the winget pkgs repo to get the GAM package added: https://github.com/microsoft/winget-pkgs/pull/133570

However it is currently failing as a popup window is raised during the MSI installation, even with any of `/quiet`, `/passive`, `/qn` switches added. 
This issue has already been raised here: https://github.com/GAM-team/GAM/issues/1292

Once the PR has been accepted in the winget-pkgs repo, I'm happy to do some work and write out + link a PR in here with an additional github workflow which would auto submit new versions to the repo :)

**Comment by adamgross42** [comment]
I don't think this is an issue anymore, as @kris6673 has been publishing packages to Winget regularly.  Can the issue be closed?
